### ISSUE TITLE
_person_group table not created on new installation

### ISSUE THREAD

**Issue by wom-bat** [tbar0970/jethro-pmm#43]
On a nice shiny new empty database, I fill in the initial user form, and then I see:

    _doQuery: [Error message: Could not execute statement]
     [Last executed query:
         CREATE TABLE person_group_headcount ( `date` DATE NOT NULL, `person_groupid`     INT(11) NOT NULL, `number` INT(11) NOT NULL, PRIMARY KEY (`date`, `person_groupid`), CONSTRAINT FOREIGN KEY (`person_groupid`) REFERENCES `_person_group` (`id`) ) Engine=InnoDB;] [Native code: 1005] [Native message: Can't create table 'jethroch_jethro.person_group_headcount' (errno: 150)]

This is because the _person_group table does not yet exist.  I can't find in the source where it is meant to be created.
```
mysql> show tables;
+--------------------------------+
| Tables_in_jethroch_jethro      |
+--------------------------------+
| abstract_note                  |
| action_plan                    |
| attendance_record              |
| congregation                   |
| congregation_headcount         |
| congregation_service_component |
| family                         |
| family_note                    |
+--------------------------------+
8 rows in set (0.00 sec)
```

**Comment by tbar0970** [comment]
Thanks for reporting.  This is a problem with the order the tables are created in.  It really needs to wait til the end before creating foreign keys. Pity I didn't spot this earlier.  Will fix shortly.

**Comment by wom-bat** [comment]
Where is the table created?  The only creating reference I can find to _person_group is in the upgrade scripts, where it's renamed from person_group on the 2011 upgrade.

**Comment by tbar0970** [comment]
It's created via person_group::getInitSQL() (db_objects/person_group.class.php).
Fix coming in the next release later today.

**Comment by tbar0970** [comment]
Fix released in v2.10.2
